.github/AL-Go-Settings.json

```diff
@@ -33,9 +33,9 @@
     "Default",
     "Clean"
   ],
-  "enablePerTenantExtensionCop":  false,
   "enableCodeCop": true, 
   "CICDPushBranches": [
     "main"
-  ]
+  ],
+  "rulesetFile": "..\\Build\\Rulesets\\module.ruleset.json"
```

Comment:
Is the _rulesetFile_ path relative to this file or to the corresponding project's folder?

In this case it wouldn't matter as all projects are directly in the root, but it might be a problem if we add projects on a deeper level.

Reply:
IMO we should elevate the settings to the highest place in the "settings hierarchy" we can. If this becomes an issue in the future we can always move it down to the project settings.

Reply:
All good then. Resolving this comment.